src/libraries/System.Private.CoreLib/src/System/Collections/Hashtable.cs

```diff
@@ -1385,7 +1382,7 @@ public override void OnDeserialization(object? sender)
                 // call OnDeserialization on our parent table.
             }
 
-            internal override KeyValuePairs[] ToKeyValuePairsArray()
+            internal override DebugViewDictionaryItem<object, object?>[] ToKeyValuePairsArray()
```

Comment:
Nit: this is no longer a "KeyValuePairsArray"

Reply:
Done.